### ISSUE TITLE
Luminosity contrast ratio of 'Link existing work item', 'a11y workshop demos' and 'Issue' icon, with background is less than minimum required ratio 3:1.

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#583]
Impact on a Person with Disability (PWD):
Users who are color sensitive and have low vision are facing difficulty to identify the control as luminosity contrast ratio of 'Link existing work item', 'a11y workshop demos' and 'Issue' icon, with background is less than minimum required ratio 3:1.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Tools: Color contrast analyzer.

Pre- Requisite:
Tools: Color contrast analyzer.

Repro Steps:

Step 1: Open MCAPS ADO in Edge browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to 'Act' and invoke it->Add work item.
Step 4: Verify whether luminosity contrast ratio of 'Link existing work item', 'a11y workshop demos' and 'Issue' icon, with background is less than minimum required ratio 3:1.

Actual Result:
Luminosity contrast ratio of 'Link existing work item', 'a11y workshop demos' and 'Issue' icon, with background is less than minimum required ratio 3:1.
Current ratio of Link existing work item: 1.2:1.
Current ratio of Issue: 1.7:1
current ratio of a11y workshop demos:1.5:1

Expected Result:
 Luminosity contrast ratio of 'Link existing work item', 'a11y workshop demos' and 'Issue' icon, with its background should be greater than or equal to minimum required ratio 3:1.

MAS Reference:
[MAS 1.4.11 – Non-text Contrast](https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7bB19D1152-56E4-44F4-832A-A650AC312C4E%7d&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&wdLOR=c775A6265-CA44-4E29-B0A5-5D8B6034205C&action=default&mobileredirect=true)

WCAG Reference:
[Understanding Success Criterion 1.4.11: Non-text Contrast (w3.org)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1714260
